src/pages/Graph/SummaryPanelGraph.tsx

```diff
@@ -77,15 +77,14 @@ export default class SummaryPanelGraph extends React.Component<SummaryPanelPropT
       return null;
     }
 
-    const baseQuery = cy
-      .nodes()
-      .filter('[!isGroup]')
-      .filter('[!isRoot]');
```

Comment:
`!isRoot` filter is removed... I remember this was to not count "unknown". But yes, there are other cases where we want to count roots...

Reply:
Seeing your comment I looked at this code again and simplified it just a bit.  Each count is now just using a dedicated cy selector based on node type.  When dealing with app nodes I explicitly filter out the group nodes.  Please take a look, I think it is more clear this way.